games-puzzle/nudoku/nudoku-0.2.5.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
No strong opinion here but were I to touch a package and saw EAPI in apostrophes, I would probably remove them. ;) Either way we usually add a linebreak after EAPI definition.